finalPrep/annotations.js

```diff
@@ -2,41 +2,64 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+//create a function named buildABear with 5 parameters
 function buildABear(name, age, fur, clothes, specialPower) {
+//create a variable named greeting to be a string that uses interpolation to include name
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+//create a variable named demographics to be an array that contains parameter name and age
   var demographics = [name, age];
+//create a variable named powerSaying to be a string and concatenate parameter specialPower
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+// create a object named builtBear with 6 key-values pair
   var builtBear = {
+    // the key is basicInfo and value is demographics
     basicInfo: demographics,
+    // the key is clothes and value is clothes
     clothes: clothes,
+    // the key is exterior and value is fur
```

Comment:
You could think about pointing out whether the value comes from a previously declared variable, or from a parameter.